python/cuml/test/test_text_feature_extraction.py

```diff
@@ -0,0 +1,246 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from cuml.feature_extraction.text import CountVectorizer
+import cupy as cp
+import pytest
+from sklearn.feature_extraction.text import CountVectorizer as SkCountVect
+from cudf import Series
+from numpy.testing import assert_array_equal
+import numpy as np
+
+
+def test_count_vectorizer():
+    corpus = [
+        'This is the first document.',
+        'This document is the second document.',
+        'And this is the third one.',
+        'Is this the first document?',
+    ]
+
+    res = CountVectorizer().fit_transform(Series(corpus))
+    ref = SkCountVect().fit_transform(corpus)
+    cp.testing.assert_array_equal(res.todense(), ref.toarray())
+
+
+JUNK_FOOD_DOCS = (
```

Comment:
We should add a empty sentence based test here.

Reply:
Agreed, I added one!